WebGUI/html/FEMacroTest.html

```diff
@@ -280,6 +280,25 @@
 			{
 				firstTime = false;
 			}
+			var windowTooltip = "Welcome to the FE Macros web app!\n\n" + 
+				"This app provides an interface for devices configured in the context. " +
+				"It allows for common functions such as resets and reading and writing to memory. . " +
+				"\n\n" +
+				"Choose a FE type (e.g., CFO, DTC, or ROC) to access functionalities. " +
+				"Alternatively, you can handpick a subset of devices for more granular control by selecting <b>FE Target UID</b> or <b>toggle Multi-select</b>. " +
+				"Once you've chosen your FE type and target(s), select the Front-End (FE) Macro name, and hit <b>Run</b> to execute. " +
+				"The FE Macro will only run one time and on the devices specified. " + 
+				"For more information about a FE Macro click on the tooltip icon below the FE Macro name. " + 
+				"\n\n" +
+				"The output is displayed in the middle panel. " +
+				"\n\n" +
+				"The panel on the right tracks your history. You can run previous FE Macros by clicking on them. " +
+				"Optionally, the <b>Live View</b> button can repeatedly run the last FE Macro for a given amount of time. " + 
+				"\n\n" +
+				"Clicking on <b>Sequence Mode</b> on the top left corner will open a fourth panel. Here, users can create an alias for a sequence of previously ran FE Macros. " 
+
+			DesktopContent.tooltip("FE Macros Introduction",windowTooltip);
+			DesktopContent.setWindowScrollTop(windowTooltip);
```

Comment:
what does setWindowScrollTop do?

Reply:
That is a typo and has been fixed. **setWindowScrollTop** is defined here https://github.com/art-daq/otsdaq_utilities/blob/jcampos/UpdateFEMacroTooltips/WebGUI/js/DesktopContent.js#L2231